app/controllers/applicants_controller.rb

```diff
@@ -286,6 +289,15 @@ def after_save_path
 
     organisation_applicant_path(@organisation, @applicant)
   end
+
+  def index_landing_path
+    structure =
+      department_level? ? Department.find(params[:department_id]) : Organisation.find(params[:organisation_id])
```

Comment:
J'ai oublié de mettre ça (même si ça ne devrait pas changer grand chose ici)
```suggestion
      department_level? ? Department.includes(:motif_categories).find(params[:department_id]) : Organisation.includes(:motif_categories).find(params[:organisation_id])
```